apps/src/applab/applab.js

```diff
@@ -557,6 +558,9 @@ Applab.init = function (config) {
   config.runButtonClickWrapper = runButtonClickWrapper;
 
   Applab.channelId = config.channel;
+  Applab.firebaseName = config.firebaseName;
+  Applab.firebaseAuthToken = config.firebaseAuthToken;
```

Comment:
added a TODO:
```
// TODO(dave): convert FirebaseStorage to an ES6 class, so that we can pass in
// firebaseName and firebaseAuthToken rather than access them as globals.
```